src/metabase/sync/analyze/fingerprint/fingerprinters.clj

```diff
@@ -224,11 +223,27 @@
         :q1  q1
         :q3  q3)))))
 
+(def truncation-size
```

Comment:
I'd move this to `fingerprint` namespace. There's now nothing in this ns that uses it.